### ISSUE TITLE
Zscaler Blocking Access to Figma API, Getting "Error: Missing one or more of required arguments: "figmaToken", "figmaUrl""

### ISSUE THREAD

**Issue by ericnguyen23** [mikaelvesavuori/figmagic#155]
**Describe the bug**
Running `figmagic` returns:

```
Attempting to fetch data from Figma API...
Error: Error: Error: Missing one or more of required arguments: "figmaToken", "figmaUrl" when attempting to get data from Figma API!
```

I'm aware there is the same issue posted and solved by removing spaces in .env file, however this one is specific to Zscaler security. I validated this by running Figma API in Postman, one with and one without attaching Zscaler PEM Certificate file. 

Question is, how to I configure Figmagic to use the PEM file?

**To Reproduce**
1. run `figmagic` (of course you'll need to be under a network security like, Zscaler)

**Expected behavior**
Successfully run `figmagic` and return tokens from Figma

**Screenshots**
<img width="660" alt="console error" src="https://user-images.githubusercontent.com/37029304/147265819-8d0705a5-f592-4358-8777-996bb1e81b70.png">

<img width="805" alt="Error" src="https://user-images.githubusercontent.com/37029304/147265829-3173015d-ecf3-4243-98df-73d40f4d8608.png">

<img width="1246" alt="Success" src="https://user-images.githubusercontent.com/37029304/147265838-3ff20399-6621-478e-a858-eb73a31c4df9.png">

**Desktop (please complete the following information):**
 - OS: macOS BigSur v11.6

**Additional context**
There's a stackoverflow thread related to Zscaler npm, hope it may help: https://stackoverflow.com/questions/23788564/npm-add-root-ca

**Comment by mikaelvesavuori** [comment]
Hey! There is no support at the moment for this. Let me try adding that after Christmas!

**Comment by ericnguyen23** [comment]
> 

Thank you, much appreciated!

**Comment by mikaelvesavuori** [comment]
Hi again. I've added a very basic implementation that I'm not really sure how to test right now. It's available in `4.4.4` and release notes can be [read here](https://github.com/mikaelvesavuori/figmagic/releases/tag/v4.4.4).

**Comment by mikaelvesavuori** [comment]
Feel free to open up or discuss; closing this for now.
